helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -134,3 +134,12 @@ dex:
 
 falco:
   enabled: true
+
+coredns:
+  enabled: true
+  externalDns:
+    ips:
+      - "0.0.0.0/0"
+
+dnsAutoscaler:
+  enabled: true
```

Comment:
I think these should be moved to `networkpolicy-common` values file instead. Same with the values file in `netowrkpolicy-workload`

Reply:
done